airbyte-integrations/connectors/source-shopify/source_shopify/utils.py

```diff
@@ -9,9 +9,21 @@
 
 import requests
 
+SCOPES_MAPPING = {
+    "read_customers": ["Customers"],
+    "read_orders": ["Orders", "AbandonedCheckouts", "Transactions", "Fulfillments", "OrderRefunds", "OrderRisks"],
```

Comment:
`read_checkouts` should correspond to `AbandonedCheckouts` stream, as of the part of `read_orders`, please correct this.

Reply:
According to [doc](https://shopify.dev/api/usage/access-scopes) `AbandonedCheckouts` correspond only to `read_orders`.